client/my-sites/checkout/checkout-thank-you/redesign-v2/utils.ts

```diff
@@ -47,7 +48,21 @@ export const isRefactoredForThankYouV2 = ( props: CheckoutThankYouCombinedProps
 	if ( purchases.length === 1 ) {
 		const purchase = purchases[ 0 ];
 
-		return isWpComPlan( purchase.productSlug ) || isP2Plus( purchase ) || isTitanMail( purchase );
+		if ( isWpComPlan( purchase.productSlug ) ) {
+			return true;
+		}
+
+		if ( isP2Plus( purchase ) ) {
+			return true;
+		}
+
+		if ( isTitanMail( purchase ) ) {
+			return true;
+		}
+
+		if ( isJetpackSearchSlug( purchase.productSlug ) ) {
+			return true;
+		}
```

Comment:
The `isRefactoredForThankYouV2()` function is used to determine whether or not to render the redesigned thank you page. We are adjusting the function here to flip over to the redesigned congrats page for Jetpack Search.